Readme.md

```diff
@@ -1,34 +1,32 @@
 # Readme
 
-This install file configures your Raspberrypi 3 Model B as a Wifi-Access Point (AP)
+This install file configures your Raspberrypi 3 as a Wifi-Access Point (AP)
 by sharing the incoming internet connection on `eth0` on the `wlan0` device
 (internal WiFi Chip on the new Pi).
 
 ## Goal
 Setup your Raspberrypi 3 as a WiFi Access Point (or Router) to share your ethernet connection with friends and family as a nice take-it-with-you WiFi Network.
 
 ## Environment
-This script was tested on 
-
+This script was tested on:
 - Raspberry-Pi 3 Model B
 - No additonal HW connected
-- Rasbian GBU/Linux 8 (`cat /etc/issue`) and Debian version 8.0 (`cat /etc/debian_version`)
+- Rasbian GBU/Linux 9 (`cat /etc/issue`) and Debian version 9.6 (`cat /etc/debian_version`)
 - No changes made to the configuration so far
 
 ## Installing
 ```
-git clone https://github.com/sebastianzillessen/WiFi-AP-Raspberry-Pi-3.git
-cd WiFi-AP-Raspberry-Pi-3
-sudo su
-chmod +x install.sh
-./install.sh
+git clone https://github.com/lavirott/wifi-ap-pi3.git
+cd wifi-ap-pi3
```

Comment:
how did you come to the new folder, when I check it out I get `WiFi-AP-Raspberry-Pi-3`

Reply:
I rebased code depending on my repository (which I renaming to avoid the use of uppercase letters). This was not supposed to be included in the pull request.

Reply:
OK, I made a rollback with the new pull request (it was an error).